docs/source/conf.py

```diff
@@ -78,6 +78,7 @@
     ("py:class", "numpy.int64"),
     # TODO - Delete this once Vivienne has merged this feature into develop
     ("py:class", "virtual_rainforest.models.abiotic.Energy_balance"),
```

Comment:
Could probably delete this line as you changed the links in your docstrings to `virtual_rainforest.models.abiotic.energy_balance.Energy_balance`, so this one isn't required any more